### ISSUE TITLE
When running phpstan from a subdirectory, allowIn is ignored

### ISSUE THREAD

**Issue by reinder83** [spaze/phpstan-disallowed-calls#100]
When you run phpstan from a subdirectory of the project the allowIn is not checked

`../../vendor/bin/phpstan --configuration=../../phpstan.neon`

```yaml
    disallowedFunctionCalls:
      - function: 'var_export()'
        message: 'use logger instead'
        allowIn:
        - tests/ExtendedFakes/ExtendedQueueFake.php
      - function: 'var_dump()'
        message: 'use logger instead'
      - function: 'print_r()'
        message: 'use logger instead'
      - function: 'dd()'
        message: 'forbidden function'
      - function: 'dump()'
        message: 'forbidden function'
        allowIn:
        - tests/ExtendedFakes/ExtendedQueueFake.php
      - function: 'exit()'
        message: 'forbidden function'
      - function: 'die()'
        message: 'forbidden function'
 ```
 
 Gives me:
 ```
  ------ ------------------------------------------------------- 
  Line   tests/ExtendedFakes/ExtendedQueueFake.php              
 ------ ------------------------------------------------------- 
  51     Calling dump() is forbidden, forbidden function        
  68     Calling var_export() is forbidden, use logger instead  
  77     Calling dump() is forbidden, forbidden function        
 ------ ------------------------------------------------------- 
```

Edit: running from the root, gives no errors

**Comment by spaze** [comment]
Hey @reinder83 and thanks for the issue. Seems like things work internally a bit differently than it seemed at the first sight. Spent some time researching options and the only one I could find doable was to add a config option to specify a root dir from where all the `allowIn` paths are resolved, which also fixes this issue. I'll release the fix shortly.

**Comment by spaze** [comment]
Released now in [2.2.0](https://github.com/spaze/phpstan-disallowed-calls/releases/tag/v2.2.0).
